.codecov.yml

```diff
@@ -1,17 +1,14 @@
 # Documentation: https://docs.codecov.io/docs/codecov-yaml
 
-codecov:
-  bot: dlang-bot
-
```

Comment:
Not sure it's seems sensible to me to enable an advanced configuration by default?
Don't they know how to deal with CI? I mean eventually they should just react to us sending sth. from a CI run.